introduction/examples/thompson/most_used.py

```diff
@@ -0,0 +1,19 @@
+from collections import Counter
+import operator
+
+
+def most_used(words, n):
+    with open(words, 'r') as myfile:
+        data = myfile.read().replace('\n', ' ')
+    words = data.split()
+    dict = Counter(words)
+    sorted_dict = sorted(dict.items(), key=operator.itemgetter(1), reverse=True)
+
+    i = 0
+    while n > 0:
```

Comment:
Zamień na pętlę `for`
Głównie chodzi o to aby w pętli na końcu nie było tych śmiesznych inkrementacji czy dekrementacji iteretorów :)